// European Union Public License version 1.2
// Copyright Â© 2023 Rick Beerendonk

package org.exceptions._throw

fun main() {
    throw Exception("HELP!")
}
